enterprise_access/settings/base.py

```diff
@@ -505,11 +507,15 @@ def root(*path_fragments):
 SIMPLE_HISTORY_DATE_INDEX = False
 
 # Cache timeouts
-SUBSIDY_RECORD_CACHE_TIMEOUT = 60 * 5
-ENTERPRISE_USER_RECORD_CACHE_TIMEOUT = 60 * 5
-SUBSIDY_AGGREGATES_CACHE_TIMEOUT = 60 * 10
-
-ALL_ENTERPRISE_GROUP_MEMBERS_CACHE_TIMEOUT = 60 * 5
+DEFAULT_CACHE_TIMEOUT = 60 * 5  # 5 minutes
+CONTENT_METADATA_CACHE_TIMEOUT = 60 * 30  # 30 minutes
+ENTERPRISE_USER_RECORD_CACHE_TIMEOUT = 60 * 10  # 10 minutes
+SUBSIDY_AGGREGATES_CACHE_TIMEOUT = 60 * 10  # 10 minutes
+SUBSCRIPTION_LICENSES_LEARNER_CACHE_TIMEOUT = 60 * 2  # 2 minutes
```

Comment:
[inform] Opted to keep the `learner-licenses` cache timeout a bit smaller to reflect potentially more frequently updated data (e.g., admin-driven license assignment/revocation, etc.), as compared to enterprise customer users or content metadata.